PYME/ui/AUIFrame.py

```diff
@@ -129,8 +129,18 @@ def CreateFoldPanel(self):
         self.Update()
         self._mgr.Update()
     
-    def select_page_by_name(self, name):
-        """set a page to be active, using just it's caption
+    def _select_page_by_name(self, name):
+        """
+        WARNING - ADDED FOR TEMPORARY FUNCTIONALITY - WILL PROBABLY BE REMOVED
+        
+        FIXME - this is likely broken for page captions with spaces (where name != caption), this includes the use case
```

Comment:
ask me how I know it wasn't broken for the use case I added it for ;)

Reply:
I thought it must have worked, but couldn't work out why it did ... looking at the AUI code it's as follows:

`_mgr.GetPaneInfo(bad_name)` returns `NonePaneInfo`

`NonePaneInfo.notebook_id == -1`

So the code as it stands will select the **last** notebook. Meaning that it will work if and only if there is only one notebook in the UI. This is usually, but not always the case.

Reply:
It works for `Simulation Settings`, `Queued Actions`, and `Chained Analysis`. On wx 4.0.4 if that matters

Reply:
If you, e.g. drag the acquisition tasks on top of the hardware panel to maximise screen estate

![image](https://user-images.githubusercontent.com/19475296/105924239-c1f59e00-60a2-11eb-91b4-4048382787e0.png)

or e.g. drag the shell and queued actions panels to a new notebook at the bottom  so you can interact with them whilst still looking at the preview image you'll run into problems. It's a bit of an edge case, but my point is that the current code relies of a quirk.

Reply:
Didn't actually know you could do that. It still worked for me in the use case you screenshot, and the `bad_name` case documented in the code, but you're right that you can derail it if you make a new notebook by dragging the shell and queued actions to the bottom, after which it doesn't work regardless of caption